docs/security/fuzzing.md

```diff
@@ -0,0 +1,37 @@
+# Fuzzing and Invariant Testing 
```

Comment:
Can we change the name of the file and title to Testing? and testing.md

its general information and guidelines for testing smart contract code. Fuzzing is a sub section